android/java/org/chromium/chrome/browser/crypto_wallet/fragments/UnlockWalletFragment.java

```diff
@@ -93,32 +93,35 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                     }
                 });
 
-        mUnlockWalletRestoreButton.setOnClickListener(v -> {
-            if (mOnNextPage != null) {
-                mOnNextPage.gotoRestorePage(false);
-                mUnlockWalletPassword.getText().clear();
-            }
-        });
+        mUnlockWalletRestoreButton.setOnClickListener(
+                v -> {
+                    if (mOnNextPage != null) {
+                        mOnNextPage.gotoRestorePage(false);
+                        mUnlockWalletPassword.getText().clear();
+                    }
+                });
 
-        mBiometricUnlockWalletImage.setOnClickListener(v -> {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P
-                    && Utils.isBiometricAvailable(getContext())) {
-                createBiometricPrompt();
-            }
-        });
+        mBiometricUnlockWalletImage.setOnClickListener(
+                v -> {
+                    if (Utils.isBiometricAvailable(requireContext())) {
+                        // noinspection NewApi
+                        createBiometricPrompt();
+                    }
+                });
 
-        if (mOnNextPage != null && mOnNextPage.showBiometricPrompt()) {
-            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P
```

Comment:
I think biometric is not available on API <=28. We should probably leave that check and show only a password way in that case.

Reply:
Correct, biometric is only available on devices running Android API 29 and upwards.
I've removed that statement as we were doing this check twice: `Utils.isBiometricAvailable(Context)` in the line below contains already a check that returns false if `VERSION.SDK < VERSION_CODES.P`.

https://github.com/brave/brave-core/blob/927cabbefb76849be5dd032adc813c14f74c7380/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java#L1028-L1032

Reply:
ok cool, I missed that!